examples/connect5.rs

```diff
@@ -651,10 +636,8 @@ fn eval(pos: &Pos, _ply: i32) -> i32 {
                     return 2560;
                 }
             }
-        } else {
-            if check_patterndead4(pos, atk) > 0 {
```

Comment:
I'm not a fan of this clippy lint, I think in many cases it actually makes the code *less* readable.

Reply:
@Amanieu which clippy lint are you not a fan of?  Most of these are simple stylistic changes that are pretty standard in the Rust code... to my knowledge

Reply:
Specifically the one that turns `else { if X {} }` into `else if X {}`. I often find that the additional nesting improves readability, especially if the `X` condition is completely unrelated to the condition in the outer `if`.

Reply:
Could you undo the change to `else` here and above?